test_app/tests/activitystream/test_api.py

```diff
@@ -17,18 +17,18 @@ def test_activitystream_api_read(admin_api_client, user):
     response = admin_api_client.get(url)
     assert response.status_code == 200
     assert response.data["count"] > 0
-    assert response.data["results"][0]["operation"] == "create"
+    assert response.data["results"][-1]["operation"] == "create"
```

Comment:
This is required just because we've reversed the order of the activity stream?
If so, should we consider altering ActivityStream to use a default ordering of created_on to not change behavior?